src/EnergyPlus/EMSManager.cc

```diff
@@ -246,7 +246,8 @@ namespace EMSManager {
 
     // MODULE SUBROUTINES:
 
-    void ManageEMS(int const iCalledFrom,                 // indicates where subroutine was called from, parameters in DataGlobals.
+    void ManageEMS(EnergyPlusData &state,
```

Comment:
This is the root of most of these changes.  Since we are now passing our callback functions a state instance, then every call to ManageEMS must now have access to the state object.  And ManageEMS is called in many nooks of the code.  So all of these code paths needed the state argument added.  No functional changes for normal EnergyPlus workflows/calculations, just passing state around.

Reply:
Fewer for @brianlball  and I to do!